docs/stake-pool-course/handbook/grafana-dashboard-tutorial.md

```diff
@@ -502,3 +502,64 @@ Now click on "Apply" and thats it! You should be able to see all your Leader Slo
 
 
 Happy minting!
+
+
+## 9. Adding crypto exchange rates to your Grafana
+
+Yes, I know you should not look into those prices all day long, but I personally find quite useful to have it in one place and no need to open my cryptoexchange account every 2 minutes.
```

Comment:
Having first person view (use of "I" ) across the section would be a bit inconsistent with the rest of the guide. Other than that looks good

Reply:
I tried my best to address your remarks, but again English is not my native language, so I hope it is much better now. Thank you!